Makefile

```
@@ -18,7 +18,7 @@ test-integ: test
 
 version ?= latest
 test-docker:
-	curl -s https://raw.githubusercontent.com/sendgrid/sendgrid-oai/HEAD/prism/prism.sh -o prism.sh
+	curl -s "https://raw.githubusercontent.com/sendgrid/sendgrid-oai/refs/heads/main/prism/prism.sh?token=$(GITHUB_TOKEN)" -o prism.sh
```

If this is a private repo, maybe token in headers would work

This also needs to be added to...a lot of sendgrid client libraries as they all use prism for testing
```suggestion
    curl -H "Authorization: token $GITHUB_TOKEN" -s "https://raw.githubusercontent.com/sendgrid/sendgrid-oai/main/prism/prism.sh" -o prism.sh
```